dora/core/client/fs/src/main/java/alluxio/client/file/FileSystemContext.java

```diff
@@ -58,6 +58,7 @@
 import com.google.common.base.Preconditions;
 import io.netty.bootstrap.Bootstrap;
 import io.netty.channel.Channel;
+import org.apache.curator.shaded.com.google.common.collect.Streams;
```

Comment:
can u use the original instead of shaded lib?

Reply:
good catch!